apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: VideoPlayer
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/VideoPlayer/VideoPlayer.tsx
+storybook: '/brand/storybook/?path=/story/components-videoplayer--playground'
+description: Use the video player component to present videos availible via CDN or local.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {VideoPlayer} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<VideoPlayer title="HLS On Demand Demo" crossOrigin="anonymous">
+  <VideoPlayer.Source src="https://cdn.api.video/vod/vi36GGvd6PoTqViQLxBWwHjJ/mp4/1080/source.mp4" />
+  <VideoPlayer.Track
+    src="https://raw.githubusercontent.com/JoshBowdenConcepts/Subtitles-Example/main/sample.vtt"
```

Comment:
This is pointing at your personal account right now, could we use a GitHub/Primer-hosted version instead please?

Reply:
Same also applies for all other instances 🙏

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/373/commits/a15991caf8ad89311382ef5e27a02f6acfcc13c7)